timeseries/src/autogluon/timeseries/models/local/statsmodels.py

```diff
@@ -143,7 +143,7 @@ def _predict_with_local_model(
 
         results = [predictions.predicted_mean.rename("mean")]
         coverage_fn = lambda alpha: predictions.pred_int(alpha=alpha)
-        results += get_quantiles_from_statsmodels(coverage_fn=coverage_fn, quantile_levels=quantile_levels)
```

Comment:
How did this work before? `quantile_levels` appears undefined

Reply:
Fixed in #3252